_posts/factorio/2025-03-24-using-the-mod.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "How to play the Randovania factorio mod"
+tags: factorio
+---
+The [Randovania mod in Factorio's mod portal](https://mods.factorio.com/mod/randovania) is just a companion to main Randovania application, which can be downloaded from [here](https://randovania.org/download/).
+
+## How to configure the mod
+
+1. Download the Randovania application from [here](https://randovania.org/download/).
+
+2. Open the application, go to Games, select Factorio.
+
+3. Click "Quick Generate"
+    
+    a. Wait for it to finish.
```

Comment:
formatting here seems weird?

Reply:
Wanted it to not be a full on new step, just a note.